### ISSUE TITLE
Going beyond 48KHz, up to 88KHz and beyond.

### ISSUE THREAD

**Issue by ARTUROSING** [haoheliu/versatile_audio_super_resolution#9]
How realistic is it to make 48KHz into 96KHz for example? And will it be like a real HI-FI sound, as if recorded in very high quality originally? If we use a usual 44KHz mp3 file as a source.

Thank you very much for your attention.

**Comment by km4sh** [comment]
+1, 44.1kHz or 48kHz is fine. but 96kHz and 192kHz are more useful when processing, editing music. looking forward to it!

**Comment by falseywinchnet** [comment]
exponentially larger training time

**Comment by ARTUROSING** [comment]
> exponentially larger training time

No, I mean whether the model is able to extrapolate to higher frequencies than 48KHz, without any re-training of the original models. For example, the same Stable Diffusion is able to do this, although with some difficulties.

**Comment by ARTUROSING** [comment]
> +1, 44.1kHz or 48kHz is fine. but 96kHz and 192kHz are more useful when processing, editing music. looking forward to it!
The fact that we can get sounds from AudioLDM 2, for example, and enhance them with AudioSR, resulting in really high quality samples is a very good thing.

But what if some musicians or recording professionals want to increase the resolution of AudioSR to 96KHz or even higher. It would be very cool to make a normal and already good to hear MP3, Hi-Res 89,96, or 192KHz.
And I have to admit that I like AudioSR a lot, I'm impressed. No technology can increase the resolution and quality of sound so well, even if its sampling rate is very low.

**Comment by falseywinchnet** [comment]
there are very few sounds with frequency higher than 48k that you can hear.
such higher resolution is more useful for mastering- which is to say,
you're not really going to get
higher resolution data, you're just going to get a higher resolution
sampling rate.
